tools/gradle-plugin/src/test/kotlin/io/github/composegears/valkyrie/gradle/Utils.kt

```diff
@@ -0,0 +1,106 @@
+@file:OptIn(ExperimentalPathApi::class)
+
+package io.github.composegears.valkyrie.gradle
+
+import assertk.Assert
+import assertk.assertions.support.fail
+import java.nio.file.Path
+import java.nio.file.Paths
+import java.util.Properties
+import kotlin.io.path.ExperimentalPathApi
+import kotlin.io.path.absolutePathString
+import kotlin.io.path.copyToRecursively
+import kotlin.io.path.createDirectories
+import kotlin.io.path.exists
+import kotlin.io.path.inputStream
+import kotlin.io.path.writeText
+import org.gradle.testkit.runner.BuildResult
+import org.gradle.testkit.runner.GradleRunner
+import org.gradle.testkit.runner.TaskOutcome
+import org.gradle.testkit.runner.TaskOutcome.SUCCESS
+
+internal fun buildRunner(root: Path, androidHome: Path? = null) = GradleRunner
+    .create()
+    .withPluginClasspath()
+    .withGradleVersion(System.getProperty("test.version.gradle"))
+    .withProjectDir(root.toFile())
+    .apply {
+        if (androidHome != null) {
+            withEnvironment(mapOf("ANDROID_HOME" to androidHome.absolutePathString()))
```

Comment:
Providing `ANDROID_HOME` in test sys properties is good enough, we must declared it explicitly here?

Reply:
Seems you can use `environment` in test blocks.